pkgs/development/python-modules/icontract/default.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+
+  # build
+, asttokens
+, typing-extensions
+
+  # tests
+, pytestCheckHook
+, mypy
+, pylint
+, yapf
+, tox
+, pydocstyle
+, coverage
+, docutils
+, pygments
+, dpcontracts
+, tabulate
+, py-cpuinfo
+, typeguard
+, astor
+, numpy
+, asyncstdlib
+}:
+
+buildPythonPackage rec {
+  pname = "icontract";
+  version = "2.6.1";
+  format = "setuptools";
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "Parquery";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
This is intentional. See [here](https://github.com/NixOS/nixpkgs/commit/7f5915b04801f383a3504a011601e2fb09dfedf8)

Reply:
That was a bug in the updater. The repo has no branch named like the tag.

Reply:
but why not use the `refs/tags` prefix? This way it is more explicit in the (unlikely) event a branch will be named like the tag in the future